server/game/PropertyTypes/ReferenceCountedSetProperty.js

```diff
@@ -6,13 +6,17 @@ class ReferenceCountedSetProperty {
     add(value) {
         let lowerCaseValue = value.toLowerCase();
         let currentCount = this.referenceCounts.get(lowerCaseValue) || 0;
-        this.referenceCounts.set(lowerCaseValue, currentCount + 1);
+        let newCount = currentCount + 1;
+        this.referenceCounts.set(lowerCaseValue, newCount);
+        return currentCount <= 0 && newCount > 0;
```

Comment:
This is outdated and can be removed.

For reference, it was when the DT was ruling reactions to icon adding or removal as when the icon itself was actually added or removed, rather than just an instance being added or removed. This code was tracking when a reference would actually be added after an instance was added by comparing it's count before and after the adjustmemt.

Reply:
This has been fixed/removed in a942bb5c0b401056a091d332f3580f7c3841603a